asttokens/mark_tokens.py

```diff
@@ -99,10 +99,7 @@ def _visit_after_children(self, node, parent_token, token):
     node.last_token = nlast
 
   def _find_last_in_line(self, start_token):
-    try:
-      newline = self._code.find_token(start_token, token.NEWLINE)
-    except IndexError:
```

Comment:
Mmmm, I have a feeling removing this `except` can easily cause an error. Why do you think it's better to remove?

Reply:
I'm removing code that never runs. `find_token` doesn't advance past the end because it has a check `and not token.ISEOF(t.type)`. We should maintain that guarantee and also trust it.